lib/src/messages/responses/topics/topic_subscription_item.dart

```diff
@@ -16,19 +14,15 @@ class TopicSubscriptionItemBinary implements TopicSubscriptionItemResponse {
   List<int> get value => _value;
 }
 
-class TopicSubscriptionItemError extends ErrorResponseBase
-    implements TopicSubscriptionItemResponse {
-  TopicSubscriptionItemError(super.exception);
-}
+class TopicSubscriptionItemNotSet implements TopicSubscriptionItemResponse {}
 
 TopicSubscriptionItemResponse createTopicItemResponse(TopicItem_ item) {
   switch (item.value.whichKind()) {
     case TopicValue__Kind.text:
       return TopicSubscriptionItemText(item.value.text);
     case TopicValue__Kind.binary:
       return TopicSubscriptionItemBinary(item.value.binary);
-    default:
-      return TopicSubscriptionItemError(UnknownException(
-          "unknown TopicItemResponse value: ${item.value}", null, null));
+    case TopicValue__Kind.notSet:
+      return TopicSubscriptionItemNotSet();
```

Comment:
I'm a little confused about what this case signifies and what SDK users are supposed to do with it . . . ?

Reply:
It was the only other possible `TopicValue__Kind` type that wasn't accounted for, but I agree it's not exactly clear what users are supposed to do with it. I can remove it and just mention it in a log statement instead for if/when it shows up.

Reply:
This stems from how protobuf works. By default, everything is optional, so its techincally possible that the subscription item is "not set" ie the proto is empty.